hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2374,7 +2374,7 @@ class Emit[C](
             PCanonicalTuple(true, et.emitType.storageType).constructFromFields(cb, region, FastIndexedSeq(et), deepCopy = false)
           }
 
-          val bufferSpec: BufferSpec = BufferSpec.defaultUncompressed
+          val bufferSpec: BufferSpec = BufferSpec.blockedUncompressed
```

Comment:
I'm a bit skeptical of this, since CDA necessarily writes things across the wire.

Reply:
CDA should use the size based spec, I think??

Reply:
Yes, we probably should just use wireSpec. I'd like to get this (more minimal) change merged and then I'll benchmark to see if changing CDA to wireSpec is egregious, and if not, I'll make the one line change.